docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -4535,6 +4535,193 @@ Now go in `Nodes` and click on a MAC address and you will see that a new tab app
 image::TAB.png[scaledwidth="100%",alt="Node TAB"]
 
 
+Monit
+~~~~~
+
+`monit` is a utility for managing and monitoring processes, files, directories and filesystems on a Unix system. Monit conducts automatic maintenance and repair and can execute meaningful causal actions in error situations. E.g. Monit can start a process if it does not run, restart a process if it does not respond and stop a process if it uses too much resources.
+
+For further reference the monit documentation is available at: https://mmonit.com/monit/documentation/monit.html
+
+Install Monit
+^^^^^^^^^^^^^
+
+The following must be done on each server of a cluster
+
+RHEL / CentOS
+----
+yum localinstall http://packetfence.org/downloads/PacketFence/RHEL6/`uname -i`/RPMS/packetfence-release-1.2-5.1.noarch.rpm -y
```

Comment:
We have more up-to-date rpm mentioned in this guide.

Reply:
I think we can now safely remove this step (installing the packetfence-release) since the original instructions came from a time where we were installing the repo manually but its not the case anymore, so if people have PacketFence installed, they have this installed.

Reply:
Agree.